tests/unit/channel_test.py

```diff
@@ -274,7 +274,7 @@ def test_process_frames_when_connectionclosed_on_dispatch(self):
 
         expect(c.dispatch).args(f0).raises(
             ConnectionClosed('something darkside'))
-        stub(c.close)  # assert not called
+        expect(c.close).times(0)
```

Comment:
@awestendorf, here is a better example: there used to be a stub with the comment here `stub(c.close)  # assert not called`, so I am thinking that instead of explaining what the code does, it might be self-documenting to simply use the mechanism that makes it explicit: `expect(c.close).times(0)` (does what it says, says what it does)